fury/tests/test_convert.py

```diff
@@ -0,0 +1,35 @@
+import os
+import pytest
+import numpy.testing as npt
+from tempfile import TemporaryDirectory
+from fury.io import load_image
+# Optional packages
+from fury.optpkg import optional_package
+matplotlib, have_matplotlib, _ = optional_package('matplotlib')
+
+if have_matplotlib:
+    import matplotlib.pyplot as plt
+    from fury.convert import matplotlib_figure_to_numpy
+
+
+@pytest.mark.skipif(not have_matplotlib, reason="Requires MatplotLib")
+def test_convert():
+    names = ['group_a', 'group_b', 'group_c']
+    values = [1, 10, 100]
+
+    fig = plt.figure(figsize=(9, 3))
+    # plt.subplot(131)
+    # plt.bar(names, values)
+    # plt.subplot(132)
+    # plt.scatter(names, values)
+    # plt.subplot(133)
+    # plt.plot(names, values)
+    # plt.suptitle('Categorical Plotting')
+    arr2 = matplotlib_figure_to_numpy(fig, transparent=True)
+
+    with TemporaryDirectory() as tmpdir:
+        fname = os.path.join(tmpdir, 'tmp.png')
+        dpi = 100
+        fig.savefig(fname, dpi=dpi, transparent=True)
+        arr1 = load_image(fname)
+        npt.assert_array_equal(arr1, arr2)
```

Comment:
Nice test!

Reply:
Thanks! 

I've (kindof) picked it up from the code you wrote 😅 
This just tests the exact way its written in test_actors.py 

But it's not redundant code. if something breaks in future. (or we add more conversion definitions) this will be useful...